fixtures/stairville/fog-m-1000.json

```diff
@@ -0,0 +1,89 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "M-Fog 1000",
```

Comment:
FWIW, the M-Fog 1000, M-Fog 1500, and M-Fog 2500 have the same manual and same DMX mapping -- they only differ in size and power.

Reply:
It makes sense to copy the fixture once it's ready to have all 4 fixtures at once.